packages/components/releaseNotes/components.md

```diff
@@ -1,6 +1,10 @@
 # @labkey/components
 Components, models, actions, and utility functions for LabKey applications and pages.
 
+### version TBD
+*Released*: TBD
+* Support tagging participant and time point columns in sample types
```

Comment:
I know that this is the name of the epic, but it doesn't really describe the changes that are in this PR. I'd suggest adding the sub bullets that you have in the PR description CHANGES section here (maybe just the first two anyway).

Reply:
Good point! Done